triggers/tests/src/test/scala/com/digitalasset/daml/lf/engine/trigger/TriggerRuleSimulationLib.scala

```diff
@@ -0,0 +1,95 @@
+// Copyright (c) 2023 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.lf.engine.trigger
```

Comment:
Intentionally placed this code in package `com.daml.lf.engine.trigger` rather than `com.daml.lf.engine.trigger.tests` - as, at some point in the future, we may want users to run this code from the Trigger CLI.